README.md

```diff
@@ -67,131 +123,55 @@ mantle codebase.
 
-## ore
-
-Related commands to launch instances on Google Compute Engine(gce)
-within the latest SDK image. SSH keys should be added to the gce project
-metadata before launching a cluster. All commands have flags that can
-overwrite the default project, bucket, and other settings.  `ore help
-<command>` can be used to discover all the switches.
-
-### ore upload
-
-Upload latest SDK image to Google Storage and then create a gce image.
-Assumes an image packaged with the flag `--format=gce` is present.
-Common usage for CoreOS devs using the default bucket and project is:
-
-`ore upload`
+### Manhole
+The `platform.Manhole()` function creates an interactive SSH session which can
+be used to inspect a machine during a test.
 
-### ore list-images
+## kolet
+kolet is run on kola instances to run native functions in tests. Generally kolet
+is not invoked manually.
 
-Print out gce images from a project. Common usage:
-
-`ore list-images`
-
-### ore create-instances
-
-Launch instances on gce. SSH keys should be added to the metadata
-section of the gce developers console. Common usage:
-
-`ore create-instances -n=3 -image=<gce image name> -config=<cloud config file>`
-
-### ore list-instances
-
-List running gce instances. Common usage:
-
-`ore list-instances`
-
-### ore destroy-instances
-
-Destroy instances on gce based by prefix or name. Images created with
-`create-instances` use a common basename as a prefix that can also be
-used to tear down the cluster. Common usage:
-
-`ore destroy-instances -prefix=$USER`
-
-## cork
-
-Cork is a tool that helps working with CoreOS images and the SDK.
-
-### cork create
-
-Download and unpack the CoreOS SDK.
-
-`cork create`
-
-### cork enter
+## ore
+Ore provides a low level interface for each cloud provider. It has commands 
+related to launching instances on a variety of platforms (gcloud, aws,
+azure, esx, and packet) within the latest SDK image. Ore mimics the underlying
+api for each cloud provider closely, so the interface for each cloud provider
+is different. `ore help <provider>` lists the available actions and options for
+each provider.
 
-Enter the SDK chroot, and optionally run a command. The command and its
-arguments can be given after `--`.
+Note, when uploading to some cloud providers (e.g. gce) the image may need to be packaged
+with a different --format (e.g. --format=gce) when running `image_to_vm.sh`
 
-`cork enter -- repo sync`
+## plume
+Plume is the Container Linux release utility. Releases are done in two stages,
+each with their own command: pre-release and release. Both of these commands are idempotent.
 
-### cork download-image
+### plume pre-release
+The pre-release command does as much of the release process as possible without making anything public.
+This includes uploading images to cloud providers (except those like gce which don't allow us to upload
+images without making them public) and running kola tests against the uploaded images.
```

Comment:
plume doesn't run kola tests.  Jenkins does, as part of the pre-release pipeline.  Just drop the latter part.